### DISCUSSION TITLE
Please add the feature from stable which allows you to exit current map by pressing alt+f4

### DISCUSSION THREAD

**Discussion by zCri** [ppy/osu#14108] [Ideas] [answered]
Title, i think it's pretty useful for example if you're playing with a tablet and have buttons disabled instead of picking up your mouse you could just use the keybind, or at least that's what i do in standard

**Comment by bdach** [comment]
I'm confused. <kbd>Alt</kbd>+<kbd>F4</kbd> *should* already be exiting the current map. The only difference between that and stable is that it will go back to the main menu of the game instead of stopping at song select. Is this the behaviour you're after?

Also, does the pause menu not help either? (<kbd>Esc</kbd>)

**Comment by peppy** [comment, marked as answer]
This has intentionally changed. You can set a binding to exit the beatmap that isn't alt-f4, if your intention isn't to exit to the main menu.
